src/configuration/configuration.ts

```diff
@@ -189,6 +189,8 @@ class Configuration implements IConfiguration {
 
   autoindent = true;
 
+  matchpairs = '';
```

Comment:
Add `['mps', 'matchpairs']` to `optionAliases` above

Reply:
I don't understand, sorry. I do not find any example of an alias (like hls <-> hlsearch) in the repo.

Reply:
ok, I see it now. this was not present in master when i created this branch.